web/services/deleteProjectsService.php

```diff
@@ -100,11 +100,30 @@
     }
 
 
-    if (count($deleteProjects) >= 1)
-        if (ProjectsFacade::DeleteProjects($deleteProjects) == -1)
-            $string = "<return service='deleteProjects'><error id='1'>There was some error while deleting the projects</error></return>";
+    if (count($deleteProjects) == 1) {
+        $operationResult = ProjectsFacade::DeleteProject($deleteProjects[0]);
+        if (!$operationResult->getIsSuccessful()) {
+            http_response_code($operationResult->getResponseCode());
+            $string = "<return service='deleteProjects'><error id='" . $operationResult->getErrorNumber() . "'>" . $operationResult->getMessage() . "</error></return>";
+        }
+    }
+    else if (count($deleteProjects) > 1) {
+        $operationResults = ProjectsFacade::DeleteProjects($deleteProjects);
+        $errors = array_filter($operationResults, function ($item) {
+            return (!$item->getIsSuccessful());
+        });
+        if ($errors) {
+            //if multiple failures, let's just return a 500
+            http_response_code(500);
+            $string = "<return service='deleteProjects'><errors>";
+                foreach ($errors as $result) {
+                    $string .= "<error id='" . $result->getErrorNumber() . "'>" . $result->getMessage() . "</error>";
+                }
+            $string .= "</errors></return>";
+        }
+    }
```

Comment:
Can we even delete multiple projects at once from the UI? I remember we had a discussion about this for the creation of projects and landed on leaving in code that handles multiples for the time being. Still, Ana has a good point that this block could be simplified either way. I think that means that the code I wrote for creating projects could similarly be simplified.

Reply:
It's the same case here, the UI cannot even delete multiple projects but the service allows it. Following the same reasoning, I left that ability in place, just in case.

It's true it can be simplified, anyway, so I'll do it. I'll submit a follow-up for the create operation.